iplap_tritoninference.py

```diff
@@ -75,8 +87,15 @@ def infer(self,input_video_path,input_audio_path,temp_dir):
             frame_w, frame_h, ref_imgs, ref_img_sketches, out_stream, input_audio_path,
             outfile_path, Nl_content, Nl_pose
         )
-
-        return outfile_path
+        # import pdb;pdb.set_trace()
+        if bgremoval:
+            output_path=os.path.join(temp_dir,"matts")
+            os.makedirs(output_path,exist_ok=True)
+            matting(outfile_path,output_path)
```

Comment:
Similar to gfpgan, initialize modenet and related models outside, the models should be initialized when class is initialized not when related function is called. This will load model everytime you call this function and add unnecessary delays.

Reply:
ok..will do it...